.changesets/docs_bnjjj_docs_standard_instr.md

```diff
@@ -0,0 +1,22 @@
+### docs(telemetry): add docs about standard instruments for subgraph ([PR #5422](https://github.com/apollographql/router/pull/5422))
```

Comment:
As it'll be in the Documentation section, I think it'll be more presentable to just say something like:

```suggestion
### Standard instrument configuration documentation for subgraphs ([PR #5422](https://github.com/apollographql/router/pull/5422))
```